src/main/groovy/no/skatteetaten/aurora/gradle/plugins/JavaApplicationTools.groovy

```diff
@@ -203,23 +213,33 @@ class JavaApplicationTools {
         }
       }
 
+      if (webFluxEnabled) {
```

Comment:
This exclusion is added to ensure that webmvc and tomcat are not on the classpath of implementation for any reason. It will always cause issues for webflux - or worse make you believe you are using a reactive environment when in fact, you are not.